### PR TITLE
Added Web3 Content

### PR DESCRIPTION
**Pull request by Prashant-Bhapkar**
Update broken link from Docker Section.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1045,3 +1046,9 @@
 * [Web Development for Beginners - A Curriculum](https://github.com/microsoft/Web-Dev-For-Beginners) - Microsoft
 * [Web Development Tutorials for Beginners](https://www.youtube.com/playlist?list=PLoYCgNOIyGAB_8_iq1cL8MVeun7cB6eNc) - LearnCode.academy
 * [Web Information Retrieval](https://www.youtube.com/playlist?list=PLAQopGWlIcya-9yzQ8c8UtPOuCv0mFZkr) - L. Becchetti, A. Vitaletti (University of Sapienza Rome)
+
+
+### Web3
+
+* [Learning Resources](https://thedailyape.notion.site/Web3-d577f781fdf945d3a198e96c55869f7d) - Darren Lau (The Daily Ape)
```
Comment (davorpa) [inline courses/free-courses-en.md:1053]: Where are the courses? Seems a collection of links

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -1045,3 +1046,9 @@
...
+* [Video Lecture From Expert](https://crater.club) - Everything about Web3
```
Comment (davorpa) [inline courses/free-courses-en.md:1054]: Where are the course? It's more like a Mooc/Platform with several channels

**Comment by Prashant-Bhapkar** [comment]
@davorpa check the updated web3 content.

Looking forward to hearing from you.

**Comment by eshellman** [comment]
also, section need to be in same order as index

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -1045,3 +1046,10 @@
 * [Web Development for Beginners - A Curriculum](https://github.com/microsoft/Web-Dev-For-Beginners) - Microsoft
 * [Web Development Tutorials for Beginners](https://www.youtube.com/playlist?list=PLoYCgNOIyGAB_8_iq1cL8MVeun7cB6eNc) - LearnCode.academy
 * [Web Information Retrieval](https://www.youtube.com/playlist?list=PLAQopGWlIcya-9yzQ8c8UtPOuCv0mFZkr) - L. Becchetti, A. Vitaletti (University of Sapienza Rome)
+
+
+### Web3
+
+* [Become a Web3 Developer](https://learnweb3.io) - Course for everyone (Basics - Advanced - Expert) level
```
Comment (davorpa) [inline courses/free-courses-en.md:1053]: It's a MOOC, move to that section

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -1045,3 +1046,10 @@
...
+* [Learning Resources](https://web3.career/learn-web3) - Learn Web3 With These Free Resources
```
Comment (davorpa) [inline courses/free-courses-en.md:1054]: Like in the past.... Has several links without relation between, so... move to Metalist section

File: courses/free-courses-en.md [hunk 5]
```diff
@@ -1045,3 +1046,10 @@
...
+* [Video Lecture - Begining to Expert](https://www.youtube.com/watch?v=gyMwXuJrbJQ&ab_channel=freeCodeCamp.org) - Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript
```
Comment (davorpa) [inline courses/free-courses-en.md:1055]: PLEASE!!! Respect resources format guidelines: `[TITLE-LINK] - CREATORS (NOTES)`

```suggestion
* [Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course](https://www.youtube.com/watch?v=gyMwXuJrbJQ) - freeCodeCamp, `tch.:` Patrick Collins
```

**Comment by Prashant-Bhapkar** [comment]
Hi @davorpa ,
Thank you for the suggestion.

Check out the recent updates!!

**Review by davorpa — APPROVED** [review]
After some changes more... now looks good to me.

@eshellman, I doubt if move Web3 to Blockchain section or leave it as it is now, with crosslinks. Let me know.

File: courses/free-courses-en.md [hunk 6]
```diff
@@ -1052,3 +1054,10 @@
 * [Web Development for Beginners - A Curriculum](https://github.com/microsoft/Web-Dev-For-Beginners) - Microsoft
 * [Web Development Tutorials for Beginners](https://www.youtube.com/playlist?list=PLoYCgNOIyGAB_8_iq1cL8MVeun7cB6eNc) - LearnCode.academy
 * [Web Information Retrieval](https://www.youtube.com/playlist?list=PLAQopGWlIcya-9yzQ8c8UtPOuCv0mFZkr) - L. Becchetti, A. Vitaletti (University of Sapienza Rome)
+
+
+### Web3
+
+> :information_source: See also &#8230; [Blockchain](#blockchain), [Solidity](#solidity)
+
+* [Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course](https://www.youtube.com/watch?v=gyMwXuJrbJQ) - freeCodeCamp, `tch.:` Patrick Collins
```
Comment (eshellman) [inline courses/free-courses-en.md:1063]:
```suggestion
* [Learn Blockchain, Solidity, and Full Stack Web3 Development with JavaScript – 32-Hour Course](https://www.youtube.com/watch?v=gyMwXuJrbJQ) -  Patrick Collins (freeCodeCamp)
```

**Pull request merged**